files/build_templates/init_cfg.json.j2

```diff
@@ -21,8 +21,8 @@
 {%- set features = [("bgp", "enabled", false, "enabled"),
                    ("database", "always_enabled", false, "always_enabled"),
                    ("dhcp_relay", "enabled", false, "enabled"),
-                   ("lldp", "enabled", false, "enabled"),
-                   ("pmon", "enabled", false, "enabled"),
+                   ("lldp", "enabled", true, "enabled"),
```

Comment:
why was timer for lldp set to true here? Isn't there a separate PR for that. https://github.com/Azure/sonic-buildimage/pull/10744

Reply:
there is, it was a mistake, fixed now